wasm/src/programs/execution.rs

```diff
@@ -92,49 +93,74 @@ impl Deref for Execution {
-
-    #[wasm_bindgen_test]
-    fn test_execution_verification() {
-        let execution = Execution::from_string(EXECUTION).unwrap();
-        let verifying_key_bytes = crate::types::native::parameters::TransferPublicVerifier::load_bytes().unwrap();
-        let verifying_key = VerifyingKey::from_bytes(&verifying_key_bytes).unwrap();
-        assert!(
-            verify_function_execution(&execution, &verifying_key, &Program::get_credits_program(), "transfer_public")
-                .unwrap()
-        );
-    }
```

Comment:
This doesn't work in the context of the texts because they can't initialize the `XmlHttpRequest` polyfill needed to download the parameters needed to prove this function.